core/src/main/resources/jenkins/security/ResourceDomainConfiguration/help-url.html

```diff
@@ -0,0 +1,41 @@
+<p>
+    Jenkins serves many files that are potentially created by users that are not fully trusted, such as files in project workspaces or archived artifacts.
+    Without this option set, Jenkins will only serve these files with the HTTP header <code>Content-Security-Policy</code> ("CSP") set to a value disallowing many modern web features to prevent cross-site scripting (XSS) attacks on Jenkins users accessing these files.
```

Comment:
```suggestion
    Without configuring a resource domain, Jenkins will serve all files with the HTTP header <code>Content-Security-Policy</code> ("CSP") set to a value disallowing many modern web features to prevent cross-site scripting (XSS) attacks on Jenkins users accessing these files.
```
As I read this, I keep tripping over "option set" and trying to figure out  if it's a set of options or the state of some option. I find that my proposed wording reads clearer without the user having to figure it out.